src/js/components/modals/ServiceScaleFormModal.js

```diff
@@ -106,6 +106,17 @@ class ServiceScaleFormModal extends mixin(StoreMixin) {
           id: service.id,
           scaleBy: parseInt(instances, 10)
         });
+      } else if (service instanceof Pod) {
+        let spec = Object.assign({}, service.getSpec().get(),
```

Comment:
Let's move this logic to the `ServiceUtil`, so that we have one method to create scaled service specs. This has the benefit that we have an easy testable and reusable method and less logic within the view components.